CMakeLists.txt

```diff
@@ -197,6 +206,13 @@ else()
   set(CMAKE_CXX_FLAGS_DEBUG   " -DDEBUG -g3 -ggdb3")
 endif()
 
+if (${QT_PORT})
+  set(CMAKE_AUTOMOC ON)
+  find_package(WinQt5Core REQUIRED)
+  add_definitions(-DQT_PORT)
```

Comment:
Why we call it QT_PORT?
The following support platform defines lokks more simlar:
OS_POSIX
OS_QNX
OS_WINDOWS
OS_WINDOWS_QT